admin/sql/CreateFKConstraints.sql

```diff
@@ -957,6 +957,11 @@ ALTER TABLE gender
    FOREIGN KEY (parent)
    REFERENCES gender(id);
 
+ALTER TABLE genre_alias
```

Comment:
You'll need an additional fks script (run on standalone servers only) that adds this.